daemons/execd/Makefile.am

```diff
@@ -43,7 +43,7 @@ cts_exec_helper_LDADD	= $(top_builddir)/lib/common/libcrmcommon.la	\
 			  $(top_builddir)/lib/cib/libcib.la		\
 			  $(top_builddir)/lib/services/libcrmservice.la	\
 			  $(top_builddir)/lib/pengine/libpe_status.la	\
-			  $(top_builddir)/lib/internal/libpacemaker-internal.la
+			  $(top_builddir)/lib/pacemaker/libpacemaker.la
```

Comment:
Actually, using `libtool` and referencing embedded libtool archives
only (non `*.la` should then be listed last and the order may matter
in this case), and when the respective `*.la` files have properly
tracked dependencies on other `*.la` without circles, this presumably
doesn't play any role here, since

Wouldn't lose a second on this if we are to stay with `libtool` for
the foreseeable future, especially if there are no circular
dependencies amongst the embedded libtool archives.

My understanding is also that listing all the transitive-over-`*.la`
libraries, without having any symbols of these directly used in the
target, is just a bloat given that these dependencies are already
properly tracked per above.

Reply:
(e.g. `meson` combined with GNU `ld` seems to mix `--start-group`
and `--end-group` to the build command structure so as to solve the
ordering problem of static linking so being overly considerate here
is rather make-work, IMHO)

Reply:
> My understanding is also that listing all the transitive-over-`*.la`
> libraries, without having any symbols of these directly used in the
> target, is just a bloat given that these dependencies are already
> properly tracked per above.

I believe you are right about this.  Perhaps it would be worth doing a commit first to minimize what's listed in _LDADD and _LIBADD everywhere?  #1701 is related to this.

Reply:
That's good to hear. In that case, I'd just try to get the ADDs that are being changed in this PR done reasonably, and not worry about the rest (since this subsumes both libpengine and libtransitioner, #1701 will be covered).